day_two_guide/topics/api_service_status.adoc

```diff
@@ -17,13 +17,14 @@ NAME                             READY     STATUS    RESTARTS   AGE
 master-api-myserver.com          1/1       Running   0          56d
 ----
 
-The API service exposes a health check, which can be queried externally using the API host name:
+The API service exposes a health check, which can be queried externally using the API host name. Both the API service and web console share the same port, usually TCP 8443 or 443, depending on the setup. This port needs to be available within the cluster and to everyone who needs to work with the deployed environment:
 
 ----
 oc get pod -n kube-system -o wide
 NAME                                               READY     STATUS    RESTARTS   AGE       IP            NODE
 master-api-myserver.com                            1/1       Running   0          7h        10.240.0.16   myserver.com/healthz
```

Comment:
Original doc printed `myserver.com/healthz` for `NODE` column, should remove `/healthz`